rpcserver.go

```diff
@@ -5066,7 +5078,153 @@ func (r *rpcServer) SubscribeChannelBackups(req *lnrpc.ChannelBackupSubscription
 	}
 }
 
-// ChannelAcceptor method stub.
+// requestInfo is used in the ChannelAcceptor bidirectional stream and
+// encapsulates the request information sent from the RPCAcceptor to the
+// RPCServer.
+type requestInfo struct {
+	chanReq      *chanacceptor.OpenChannelRequest
+	responseChan chan bool
+}
+
+// ChannelAcceptor dispatches a bi-directional streaming RPC in which
+// OpenChannel requests are sent to the client and the client responds with
+// a boolean that tells LND whether or not to accept the channel. This allows
+// node operators to specify their own criteria for accepting inbound channels
+// through a single persistent connection.
 func (r *rpcServer) ChannelAcceptor(stream lnrpc.Lightning_ChannelAcceptorServer) error {
-	return nil
+
+	// Locate the ChainedAcceptor within the fundingmanager.
+	openChanPredicate := r.server.fundingMgr.cfg.OpenChannelPredicate
```

Comment:
instead of acccessing the funding config here, you can store the predicate as a member of the rpc server. Then you don't need the type assertion below either.

Reply:
Lmk what you think about my approach